app/src/main/java/ch/epfl/culturequest/backend/tournament/tournamentobjects/ArtQuiz.java

```diff
@@ -2,23 +2,54 @@
 
 
 import java.util.ArrayList;
+import java.util.HashMap;
 
 // Represents a full quiz composed of several questions
 public class ArtQuiz {
 
     ArrayList<QuizQuestion> questions;
 
+    HashMap<String, Integer> scores = new HashMap<>();
+
+    String artName;
+
```

Comment:
As quizzes now have the art name as an attribute, the Map<String, Quiz> - mapping artName to quizzes -, which is an attribute of the Tournament class I initially made and that you didn't modify, appears redundant and is not of high utility. Anyway, I just wanted to point it out to not make the situation more confusing as it is now for a potential future reader of our code, but if everything works well now and considering that this last sprint is over, it just can be a small technical debt that we can afford to deal with.

Reply:
We need both to make the artname accessible through the screens, in the same way we store posts with ids with a mapping from their id.